gen-lids-image-tar

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#Staging directory
+LID_STAGING_DIR="/var/lib/phosphor-software-manager/hostfw/staging"
+#Directory where the lid files without a header are stored
+LID_DIR_PATH=$LID_STAGING_DIR/"lid"
+#Directory where the image files are stored as they are built
+IMAGE_DIR_PATH=$LID_STAGING_DIR/"image"
+#Directory where the code update tarball files are stored
+UPDATE_DIR_PATH=$LID_STAGING_DIR/"update"
+#The file name of the code update tarball
+TAR_IMAGE_NAME="image.tar"
+#The file name of the hostfw image
+HOSTFW_IMAGE_NAME="image-hostfw"
+#The path to the hostfw image
+HOSTFW_IMAGE_PATH=$IMAGE_DIR_PATH/$HOSTFW_IMAGE_NAME
+#The path to the tarball file expected by the phosphor software manager
+UPDATE_IMAGE_PATH="/tmp/images/$TAR_IMAGE_NAME"
+#The path to the code update tarball file
+TAR_IMAGE_PATH=$IMAGE_DIR_PATH/$TAR_IMAGE_NAME
+
+#Create the hostfw squashfs image from the LID files without header
+
+echo Create the hostfw squashfs image from the LID files without header
+echo "Executing mksquashfs $LID_DIR_PATH $HOSTFW_IMAGE_PATH -all-root -no-recovery -no-xattrs -noI -mkfs-time 0 -all-time 0"
+
+mksquashfs $LID_DIR_PATH $HOSTFW_IMAGE_PATH -all-root -no-recovery -no-xattrs -noI -mkfs-time 0 -all-time 0
+error_code=$?
```

Comment:
No need to check for exit codes with "set -eo pipefail". PLDM wasn't deleting the directories on error so we're ok leaving the directories on error.

Reply:
done